_posts/en/2021-06-08-working-in-the-open-part-2.md

```diff
@@ -0,0 +1,172 @@
+
+If your team isn't fully ready to work in the open, try slowly opening up your workspace to the rest of the organization by default.
+Don't wait until "that presentation is ready" before letting it be discoverable by others.
+You know you will not get around to it!
+
+## Set up your Environment
+
+Your virtual workspace may consist of a variety of tools and platforms.
+
+Software developers and IT folks may leverage social coding platforms such as Gitlab, Github, GCcode, etc., to collaborate widely with people around the world on source code.
+
+In the Government of Canada, most departments now use Microsoft 365 to varying degrees, which may include Teams, a collaboration platform that ties in multiple products and capabilities.
+
+Whether you are using the corporate collaboration platform or a 3rd party software as service to work in the open, there are a number of things to keep in mind.
+
+### Stay in Control
+
+Your workspace is a very important place: it contains your work and you have a responsibility to ensure its properly secured and controlled.
+However, that doesn't mean you can't open up securely.
+
+Proper access management is required so that we can separate the two main types of users: your team's members and external contributors.
+The latter could be a person from within your organization or from the public.
+
+There may be more types of roles available depending on the platforms, but these are the main buckets we think of when working in the open.
+
+Depending on the platform you use, various controls may be offered to let you openly share your documents and your work while remaining in control of its content.
+
+In SharePoint, a platform most often available as "Inner Source" only, a team can open up their workspace by default to their colleagues and create specific folders with more restricted access to privately work on documents only when absolutely required.
+
+With most collaboration platforms, whether it's SharePoint, Google Documents, etc., there usually is an option to let external contributors consult your documents and add comments but prevent them from making changes directly.
+This way, you remain in control of the main version but you enable people to stop by and provide feedback on a continuous basis.
+
+### Ensure Privacy
+
+Public servants who work with citizen data must take great care regarding the privacy of that data and never share it publicly.
+Working in the open by default is not a concept that applies when working on a citizen's files and data.
+
+But it doesn't mean that even these teams can't work openly on improving their processes and sharing how they overcomed challenges.
+As public servants, we should continuously strive to improve our processes, look for new ways of solving issues, and share the results of our researches, as long as they are unclassified.
```

Comment:
'Research' is always singular. So '...share the results of our research'.

Reply:
Thanks!